data/simulations/FAH_SARS-CoV-2_spike.yml

```diff
@@ -0,0 +1,32 @@
+type: md
+title: Trajectories of the SARS-CoV-2 spike protein from Folding@Home
+description: All-atom MD simulations of the SARS-CoV-2 spike protein, simulated using Folding@Home. Simulations were run using either GROMACS or OPENMM and are stored as compressed binary XTC files. The dataset comprises 3 projects, each having a RUN*/CLONE*/result* directory structure. Each RUN represents a unique starting conformation, each CLONE is a unique MD run from the specified starting conformation, and each result is a fragment of the contiguous simulation.
```

Comment:
You want to do something like this with [multi-line YAML](https://yaml-multiline.info/):
````yaml
description: >
  All-atom MD simulations of the SARS-CoV-2 spike protein, simulated using Folding@Home. 
  Simulations were run using either GROMACS or OPENMM and are stored as compressed binary XTC files. 
  The dataset comprises 3 projects, each having a RUN*/CLONE*/result* directory structure. 
  Each RUN represents a unique starting conformation, each CLONE is a unique MD run from the 
  specified starting conformation, and each result is a fragment of the contiguous simulation.
  
  *Longest trajectory:* To retrieve the longest trajectory, you can download it directly from [this link](http://fah-public-data-covid-19-cryptic-pockets.s3-website-us-east-1.amazonaws.com/SARS-CoV-2/spike/analyzed/trajectory.xtc).

  *Entire dataset:* To get the whole dataset (7 TB), you can use the [AWS CLI](https://aws.amazon.com/cli/):
   ```bash
   aws s3 --no-sign-request sync s3://fah-public-data-covid-19-cryptic-pockets/SARS-CoV-2/spike/PROJ14217 .
   aws s3 --no-sign-request sync s3://fah-public-data-covid-19-cryptic-pockets/SARS-CoV-2/spike/PROJ14253 .
   aws s3 --no-sign-request sync s3://fah-public-data-covid-19-cryptic-pockets/SARS-CoV-2/spike/PROJ14561 .
   ```
````    
It would be great if you could also share distilled datasets, like clusters, or suggest how they can download just 1, 10, or 100 trajectories that have been processed into long contiguous trajectories. If you don't know how to do this, we can show you how to use our [fahmunge](https://github.com/choderalab/fahmunge) pipeline for knitting trajectories together.

Reply:
This is a good suggestion! I'll make sure to beef up the description. I have clustered models with a corresponding MSM that I plan to share with the datasets. I can update the description to link to these files.

I have initially just loaded the raw data but can go back when there is less of a time crunch to process the trajectories for each project. I can link to your fahmunge if that would be helpful to others!